.landscaper/gardenlet/blueprint.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+jsonSchema: "https://json-schema.org/draft/2019-09/schema" # required
+
+# imports are filled via the Installation CRD specifying an import section.
+imports:
+- name: runtimeCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: gardenCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: imageVectorOverwrite
+  schema:
+    type: string
+- name: componentImageVectorOverwrites
+  schema:
+    type: string
+- name: seedBackup
+  schema:
+    type: object
+    properties:
+      name:
+        type: string
+#       provider specific credentials just parsed as string and created as secret
+      providerCredentials:
+        type: object
+- name: revisionHistoryLimit
+  schema:
+    type: number
+- name: resources
+  schema:
+    type: object
+    properties:
+      requests:
+        type: object
+        properties:
+          cpu:
+            type: string
+          memory:
+            type: string
+      limits:
+        type: object
+        properties:
+          cpu:
+            type: string
+          memory:
+            type: string
+
+# check example/20-componentconfig-gardenlet.yaml for what can be configured
+- name: componentConfiguration
+  schema:
+    type: object
+
+deployExecutions:
+- name: default
+  type: GoTemplate
+  template: |
+    deployItems:
+    - name: deploy
+      type: landscaper.gardener.cloud/container
+      config:
+        apiVersion: container.deployer.landscaper.gardener.cloud/v1alpha1
+        kind: ProviderConfiguration
+
+        blueprint:
+          ref:
+            repositoryContext:
+    {{ toYaml ( index .cd.component.repositoryContexts (add (len .cd.component.repositoryContexts) -1) ) | indent 10 }}
+            componentName: {{ .cd.component.name }}
+            version: {{ .cd.component.version }}
+            resourceName: blueprint
+
+        importValues:
+          apiVersion: imports.gardenlet.landscaper.gardener.cloud/v1alpha1
+          kind: LandscaperGardenletImport
+    {{ toYaml .imports | indent 6 }}
```

Comment:
```suggestion
{{ toYaml .imports | indent 10 }}
```

Reply:
@schrodit I cannot test this from my side - can you check if that makes sense?

Reply:
This worries me a little bit :( How should we maintain this piece of code/configuration if we cannot test it/cannot be sure that what we introduce is correct?
@schrodit can you please elaborate how to test such things during development?

Reply:
you can test the rendering of the blueprint (deploy items) with the landscaper cli 
```
lsc blueprints render ./.landscaper/gardenlet \ # specify the blueprint to render
-f ./.landscaper/gardenlet/example/landscaper-resources/imports.yaml \ # example import values
-c ./.landscaper/gardenlet/example/landscaper-resources/component_descriptor.yaml # example component descriptor
```

We are also currently working on a fully local test environment so that the full flow can be tested with one command.
Adding @In-Ko as new landscaper-cli skipper and @hendrikKahl and @MrBatschner as they are working on the local testing topic.